public/app/features/dashboard-scene/edit-pane/DashboardEditPaneSplitter.tsx

```diff
@@ -26,6 +26,7 @@ export function DashboardEditPaneSplitter({ dashboard, isEditing, body, controls
   const { editPane } = dashboard.state;
   const styles = useStyles2(getStyles, headerHeight ?? 0);
   const [isCollapsed, setIsCollapsed] = useEditPaneCollapsed();
+  const [overrideEditPaneToggle, setOverrideEditPanelToggle] = useState(false);
```

Comment:
not sure we need this override, I think if you select something in edit mode we can just unhide it?

Reply:
the problem is when do we close it after opening? but maybe we don't need that?

with that flag we can also make sure that deselecting objects closes back the pane if it was initially closed. If we manually open the pane then selecting/deselecting does not influence pane visibility, it will always stay open. But if the pane is collapsed then it will open and close based on whether there is a selection or not.

Reply:
@mdvictor yea, your right.